### PR TITLE
Change message and update image size

### PR DESCRIPTION
**Pull request by sahiljoshi515**

### CODE REVIEW

**Comment by powellsl99** [comment]
@sahiljoshi515 can you change this to go into frontend-dev instead of master

**Comment by sahiljoshi515** [comment]
Base changed, ready to be merged!

File: Client/src/Pages/NotFound.js [hunk 1]
```diff
@@ -11,10 +11,10 @@ export default function notFound() {
       <br/>
       <br/>
       <Typography component="h1" variant="h5" align="center">
-        Page not found.
+        We are sorry that we couldn't find the page. Please try again.
```
[the remark this answers is not in the document]
Comment (sahiljoshi515) [reply Client/src/Pages/NotFound.js:14]: I think @powellsl99 meant to add something like the "Github 404" pages, hence we changed the text.

[the remark this answers is not in the document]
Comment (ishchhabra) [reply Client/src/Pages/NotFound.js:14]: You should atleast include the error code (404) on the page (github does that as well).

[the remark this answers is not in the document]
Comment (ishchhabra) [reply Client/src/Pages/NotFound.js:14]: Something like <h(n)>404</h(n)><h(n-1)>We are sorry that we couldn't find the page. Please try again.</h(n-1)>

[the remark this answers is not in the document]
Comment (powellsl99) [reply Client/src/Pages/NotFound.js:14]: @sahiljoshi515 I agree about adding the error code, think you could add that real quick just so we can merge in?

[the remark this answers is not in the document]
Comment (sahiljoshi515) [reply Client/src/Pages/NotFound.js]: @powellsl99 used i18n and is ready to get merged!

**Pull request merged**